content/flux/v0.x/data-types/dynamic.md

```diff
@@ -0,0 +1,622 @@
+4.  Use [`array.map()`](/flux/v0.x/stdlib/array/map/) to iterate over each element
+    in the array and explicitly [cast dynamic types to basic types](#basic-types).
+
+{{< expand-wrapper >}}
+{{% expand "View example with an array of scalar values" %}}
+
+##### JSON array of scalar values
+
+```json
+["3", 2, true]
+```
+
+##### Convert a JSON array of scalar values to a Flux array
+
+```js
+import "array"
+import "experimental/dynamic"
+
+json = jsonBytes // Byte-encoded JSON above
+parsed = dynamic.jsonParse(data: json)
+
+parsed
+    |> dynamic.asArray()
+    |> array.map(fn: (x) => int(v: x))
+
+// Returns [3, 2, 1]
+```
+{{% /expand %}}
+
+{{% expand "View example with an array of JSON objects" %}}
+
+##### JSON array of objects
+
+```json
+[
+    {"first-name": "John", "last-name": "Doe", "age": 42},
+    {"first-name": "Jane", "last-name": "Doe"},
+    {"first-name": "James", "last-name": "Doe", "age": 15}
+]
+```
+
+##### Convert a JSON array of objects to a Flux array
+
+```js
+import "array"
+import "experimental/dynamic"
+
+json = jsonBytes // Byte-encoded JSON above
+parsed = dynamic.jsonParse(data: json)
+
+parsed
+    |> dynamic.asArray()
+    |> array.map(
+        fn: (x) => ({
+            fname: string(v: x["first-name"]),
+            lname: string(v: x["last-name"]),
+            age: int(v: x.age)
+        }),
+    )
+
+// Returns [
+//     {age: 42, fname: John, lname: Doe},
+//     {age: <null>, fname: Jane, lname: Doe},
+//     {age: 15, fname: James, lname: Doe}
+// ]
+```
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+{{% /tab-content %}}
+{{% tab-content %}}
+
+**To convert a JSON object to a Flux record:**
+
+1.  Import the [`experimental/dynamic` package](/flux/v0.x/stdlib/experimental/dynamic/)
+2.  Use [`dynamic.jsonParse()`](/flux/v0.x/stdlib/experimental/dynamic/jsonparse/)
+    to parse a byte-encoded JSON string into a dynamic type.
+3.  Define a new record and [cast each property of the dynamic type to a basic type](#basic-types).
+
+##### JSON object
+
+```json
+{"first-name": "John", "last-name": "Doe", "age": 42}
+```
+
+##### Convert a JSON object to a Flux record
+
+```js
+import "experimental/dynamic"
+
+json = jsonBytes // Byte-encoded JSON above
+parsed = dynamic.jsonParse(data: json)
+
+newRecord = {
+    fname: string(v: parsed["first-name"]),
+    lname: string(v: parsed["last-name"]),
+    age: int(v: parsed.age)
+}
+
+// newRecord returns {age: 42, fname: John, lname: Doe}
+```
+{{% /tab-content %}}
+{{< /tabs-wrapper >}}
+
+
+### Include dynamic types in a table
+
+**Dynamic types are not supported as column values in Flux tables.**
+To include a dynamic value in a Flux table,
+[cast the dynamic type to a Flux basic type](#basic-types).
+
+The following example uses [`array.from()`](/flux/v0.x/stdlib/array/from/) to 
+build and ad hoc table using dynamic types.
+Each dynamic type is cast to a Flux basic type when defining the row record.
```

Comment:
> Each dynamic type **must be** cast to a Flux basic type when defining the row record.

Weak rephrase suggestion here, just to reinforce the whole idea of _the boundary_ I mentioned previously.